.scripts/compare-database.js

```diff
@@ -1,11 +1,15 @@
-import pg from 'pg';
-import assert from 'node:assert';
+import pg from "pg";
+import assert from "node:assert";
```

Comment:
Why double quotes?

Reply:
Updated with single quotes. It was the default code formatter of vscode.